mods/taskbar-clock-customization.wh.cpp

```diff
@@ -76,9 +76,12 @@ Only Windows 10 64-bit and Windows 11 are supported.
 - WebContentsMaxLength: 28
   $name: Web content maximum length
   $description: Longer strings will be truncated with ellipsis
-- WebContentsUpdateInterval: 10
+- WebContentsUpdateInterval: "0.5"
```

Comment:
I don't like that it becomes a string. It would be nice to have Windhawk support floating point numbers (probably by showing them as numbers and saving them as strings), but for now, it's not supported.

In this case, I don't think an update interval of less then 1 minute is very interesting, but a rotation interval is. So I'd add another option for rotation interval which will be a number of seconds.